qiskit/opflow/list_ops/list_op.py

```diff
@@ -75,7 +75,12 @@ def __init__(
         """
         super().__init__()
         self._oplist = self._check_input_types(oplist)
-        self._combo_fn = combo_fn
+        if combo_fn is None:
```

Comment:
Since we essentially only access the `combo_fn` through the getter, we could leave `self._combo_fn` as `None` and return the identity function in the getter in that case. That would avoid the introduction of an additional variable.